src/ImageProcessorCore/Formats/Tiff/ValueDecoders/TiffURationalDecoder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace ImageProcessorCore.Formats.Tiff.ValueDecoders
+{
+    public class TiffURationalValueDecoder : ITiffValueDecoder
+    {
+        public bool DecodeValue(TiffReader reader, IFDEntry entry, TiffTagValue value)
+        {
+            if (entry.FieldType != IFDEntryType.URational)
+                return false;
+            
+            // URATIONAL = 2 (4-byte) unsinged integers
+            var size = entry.ValueCount * 8;
+           
+            // first 4 bytes are the numerator
+            // second 4 bytes are the denominator
+            // must be an array of shorts to get here, because a single short
+            // size would be less the 4 ( or fit into the offset )
+            
+            reader.Seek(entry.ValueOffset);
+
+            var numerator = reader.GetInt32();
+            var denominator = reader.GetInt32();
+
+            // TODO: strongly type this... for now just use an anonymouse type
+            value.Value = new {Numerator = numerator, Denominator = denominator};
```

Comment:
I have a `Rational`class here 

https://github.com/JimBobSquarePants/ImageProcessor/blob/bbe67057bd08733c0a1a7b3e5a559e7d576a8c02/src/ImageProcessor/Imaging/MetaData/Rational.cs

I think we can vastly simplify it though using the new [BigInteger](https://msdn.microsoft.com/en-us/library/system.numerics.biginteger(v=vs.110).aspx) type from `System.Numerics`